content/blog/container-net-basics.md

```diff
@@ -0,0 +1,416 @@
+
+---
+title: "A Container Is a Linux Namespace and Networking Basics"
+featured: true
+description: >
+    Demystify containers and cover some networking basics
+date: 2020-26-21
+patterns:
+- Deployment
+# Author(s)
+team: 
+- Ivan Tarin
+---
+
+# A Container Is a Linux Namespace and Networking Basics
+
+![open source image @freddygthatsme see sources
+](https://raw.githubusercontent.com/everythingeverywhere/tanzu-dev-portal/main/static/images/blogs/container-net-basics/freddy-g-1i1zPbkQJNg-unsplash.jpg)
```

Comment:
the image links should be like `/images/blogs/container-net-basics/freddy-g-1i1zPbkQJNg-unsplash.jpg`